### PR TITLE
Add client and servers co-testing

### PR DESCRIPTION
**Pull request by Tricky511**
install deps:
`make install_deps`

build:
`make build`

test:
`make test`

### CODE REVIEW

**Comment by uran0sH** [comment]
Can you also support using Makefile at the same time?

**Comment by uran0sH** [comment]
LGTM. Any other problems? @luanshaotong

File: client_run.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+function green_font() {
+    echo -e "\033[32m$1\033[0m\c"
+}
+
+echo "start fuse_client_run"
+sleep 3
+mkdir ~/fs
+./target/debug/client ~/fs &
```
Comment (luanshaotong) [inline client_run.sh:17]: Where does client close?

Comment (Tricky511) [reply client_run.sh:17]: function `finish`

**Review by luanshaotong — APPROVED** [review]
LGTM

**Pull request merged**